getting-started/templates/systemlink-values.yaml

```diff
@@ -694,35 +694,45 @@ dataframeservice:
   ## Accepts units in "MiB" (Mebibytes, 1024 KiB) or in "MB" (Megabytes, 1000 KB)
   requestBodySizeLimit: 256MiB
 
-  ## Configure S3 access.
+  ## Configuration for data table row storage.
   ##
-  s3:
-    auth:
-      ## Name of the secret containing the S3 login credentials
-      ##
-      secretName: "nidataframe-s3-credentials"
-    ## The name of the S3 bucket that the service should connect to.
-    ##
-    bucket: &dataframeBucket "systemlink-dataframe"
-    ## This should just be the name of the scheme, without the trailing ://.
-    ##
-    schemeName: *s3Scheme
-    ## Set this value to connect to an external S3 instance.
-    ##
-    host: *s3Host
-    ## Set this value to connect to an S3 instance which is internal to the cluster. Ignored if host is set.
-    ##
-    service: *s3ServiceName
-    ## S3 port number.
-    ##
-    port: *s3Port
-    ## Maximum number of concurrent connections to S3 per replica.
+  storage:
+    ## The storage backend to use: currently only "s3". Defaults to s3.
+    ## s3: AWS S3 (or compatible backend such as MinIO). Additional configuration is required.
+    ##     See the "s3" section.
     ##
-    maximumConnections: 32
-    ## S3 Region
-    # <ATTENTION> This must be set to the region of the S3 instance.
+    type: s3
+
+    ## Configuration for the S3 backend. Only applicable when storage.type is "s3".
```

Comment:
Suggestion: The configuration for the S3 backend. This configuration is only applicable when the storage.type value is set to "s3".